### DISCUSSION TITLE
Linstor GUI (linstor-gui) getting started

### DISCUSSION THREAD

**Discussion by cf-sewe** [piraeusdatastore/piraeus-operator#746] [Q&A] [answered]
Could anybody provide me some starting points, how I would get [linstor-gui](https://github.com/LINBIT/linstor-gui) working with Piraeus managed linstor? 

Is the GUI meanwhile included in the piraeus-operator release / images?
Or would I need to create my own container and deployment for the GUI?

**Comment by WanzenBug** [comment, marked as answer]
It's available in the images since this commit: https://github.com/piraeusdatastore/piraeus/commit/3bae14e03be62a756a7f1b767406e7e94f6d3bf3

So you can try `kubectl port-forward services/linstor-controller api` and open http://localhost:3370/ui/ in your browser

**Reply by cf-sewe** [reply]
Thank you. I wasnt able to find out whether or not this commit is included in the latest piraeus-operator 2.7.1 release. 
Just out of curiousity, I would like to understand how the release bundling takes place in that regard?

**Reply by WanzenBug** [reply]
We rebuild the then-current release on every change to the [piraeusdatastore/piraeus](https://github.com/piraeusdatastore/piraeus/tree/master/dockerfiles) dockerfiles. So on a new Operator release, we update the configuration so it deploys the latest images [here](https://github.com/piraeusdatastore/piraeus-operator/blob/v2/config/manager/0_piraeus_datastore_images.yaml)

As you can see, we do not pin to a specific image digest, just the general software version. This means we can rebuild these images (for example when there are known CVEs), without having to make a new Operator release. But it also means that sometimes people who have already pulled the old version will not get the latest version unless they manually pull.

**Reply by cf-sewe** [reply]
That makes sense for me. Thanks a lot for the additional insight!
